myvocabularyteacher.rb

```diff
@@ -69,6 +70,11 @@ def start_search(language, word)
     @definition_word = definition_word
     pp definition_word
   end
+
+  def toggle
+    language = @current_language
+    @current_language = (language == "en_US" ? "es" : "en_US")
```

Comment:
As you are only referring to a variable without making changes, line 75 would not be needed

Reply:
Yeah, I had doubts about the assignment of the variable together with the ternary operator, that's why I added that extra variable. Even with that the first time I call toggle it doesn't seem to change the language. That's something I have to look up.